pkg/controllers/user/resourcequota/resource_quota_common.go

```diff
@@ -69,7 +70,62 @@ func convertProjectResourceLimitToResourceList(limit *v3.ResourceQuotaLimit) (co
 	return limits, nil
 }
 
-var conversion = map[string]string{
+func convertContainerResourceLimitToResourceList(limit *v3.ContainerResourceLimit) (corev1.ResourceList, corev1.ResourceList, error) {
+	in, err := json.Marshal(limit)
+	if err != nil {
+		return nil, nil, err
+	}
+	limitsMap := map[string]string{}
+	err = json.Unmarshal(in, &limitsMap)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	if len(limitsMap) == 0 {
+		return nil, nil, nil
+	}
+	logrus.Errorf("limits map is %v", limitsMap)
```

Comment:
logging in error?

Reply:
shhh, added for debug purpose :) will remove